### PR TITLE
Password feature

### PR DESCRIPTION
**Pull request by siddontang**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM.

File: pkg/proxy/router/helper.go [hunk 1]
```diff
@@ -130,8 +138,8 @@ func handleSpecCommand(cmd string, keys [][]byte, timeout int) ([]byte, bool, bo
 		shouldClose = true
 	case "SELECT":
 		b = OK_BYTES
-	case "AUTH":
-		b = OK_BYTES
+	// case "AUTH":
```
Comment (xiang90) [inline pkg/proxy/router/helper.go]: remove the commented out code?

File: pkg/proxy/router/helper_test.go [hunk 2]
```diff
@@ -192,7 +192,7 @@ func TestHandleSpecCommand(t *testing.T) {
 		"PING":   "+PONG\r\n",
 		"QUIT":   string(OK_BYTES),
 		"SELECT": string(OK_BYTES),
-		"AUTH":   string(OK_BYTES),
+		//		"AUTH":   string(OK_BYTES),
```
Comment (xiang90) [inline pkg/proxy/router/helper_test.go]: same here

**Pull request merged**